modules/kmm-deploying-modules.adoc

```diff
@@ -0,0 +1,199 @@
+
+* that match the ``Module``'s `.spec.selector`;
+* on which the kernel module is loaded (where the ModuleLoader pod has the `Ready` condition).
+
+== Creating the `Module` Custom Resource Definition
+
+The `Module` Custom Resource Definition (CRD) represents a kernel module that should be loaded on all or select nodes in the
+cluster, through a ModuleLoader image.
+A `Module` CRD specifies one or more kernel versions with which it is compatible, and a node selector.
+
+The compatible versions for a `Module` are listed under `.spec.moduleLoader.container.kernelMappings`.
+A kernel mapping can either match a `literal` version, or use `regexp` to match many of them at the same time.
+
+The reconciliation loop for `Module` runs the following steps:
+
+. list all nodes matching `.spec.selector`;
+. build a set of all kernel versions running on those nodes;
+. for each kernel version:
+ .. go through `.spec.moduleLoader.container.kernelMappings` and find the appropriate container image name.
+If the kernel mapping has `build` or `sign` defined and the container image does not already exist, run the build
+and / or signing job as required;
+ .. create a ModuleLoader `DaemonSet` with the container image determined at the previous step;
+ .. if `.spec.devicePlugin` is defined, create a device plugin `DaemonSet` using the configuration specified under
+`.spec.devicePlugin.container`;
+. garbage-collect:
+ .. existing `DaemonSets` targeting kernel versions that are not run by any node in the cluster;
+ .. successful build jobs;
+ .. successful signing jobs.
+
+== Security and permissions
+
+Loading kernel modules is a highly sensitive operation.
+Once loaded, kernel modules have all possible permissions to do any kind of operation on the node.
+
+=== `ServiceAccounts` and `SecurityContextConstraints`
+
+KMM creates privileged workload to load the kernel modules on nodes.
+That workload needs `ServiceAccounts` able to use the `privileged` `SecurityContextConstraint` (SCC).
+
+The authorization model for that workload depends on the `Module` namespace, as well as its spec:
+
+* if the `.spec.moduleLoader.serviceAccountName` or `.spec.devicePlugin.serviceAccountName` fields are set, they are
+always used;
+* if those fields are not set, then:
+ ** if the `Module` is created in the operator's namespace (`openshift-kmm` by default), then KMM will use its
+default, powerful `ServiceAccounts` to run the DaemonSets;
+ ** if the `Module` is created in any other namespace, then KMM will run the DaemonSets as the namespace's `default`
+`ServiceAccount`, which cannot run privileged workload unless you manually allow it to use the `privileged` SCC.
+
+NOTE
+!!! warning "``openshift-kmm`` is a trusted namespace"
+
+ When setting up RBAC permissions, keep in mind that any user or ServiceAccount creating a `Module` resource in the
+ `openshift-kmm` namespace will result in KMM automatically running privileged workload on potentially all nodes in
+ cluster.
```

Comment:
It looks like the conversion tool did not interpret the admonition correctly.

Our docs:

![image](https://user-images.githubusercontent.com/4248434/220860187-3b14c8dc-edda-418b-ad35-ec440dcaf383.png)

Preview:

![image](https://user-images.githubusercontent.com/4248434/220860273-a36056b0-ab03-4764-88eb-d9c53f14e1ad.png)